src/neo/Network/P2P/Payloads/Block.cs

```diff
@@ -9,19 +9,61 @@
 
 namespace Neo.Network.P2P.Payloads
 {
+    /// <summary>
+    /// Represents a block.
+    /// </summary>
     public sealed class Block : IEquatable<Block>, IInventory
     {
+        /// <summary>
+        /// The head of the block.
+        /// </summary>
         public Header Header;
+
+        /// <summary>
+        /// The transaction list of the block.
+        /// </summary>
         public Transaction[] Transactions;
 
         public UInt256 Hash => Header.Hash;
```

Comment:
```suggestion
        /// <summary>
        /// The hash of the header.
        /// </summary>
        public UInt256 Hash => Header.Hash;
```

Reply:
There is already a comment on the `IVerifible` interface.